app/views/admin/accounting/transactions/_show_modal_content.slim

```diff
@@ -0,0 +1,24 @@
+div.modal-header
+  .actions.txn-actions
```

Comment:
No transaction actions should be in the modal at this time. Edit and delete are a separate feature.